gqueries/output_elements/output_series/user_uploaded_profiles/agriculture_heat_curve.gql

```diff
@@ -22,7 +22,7 @@
         V(
           FILTER(
           ALL(),
-          "hydrogen && (hydrogen.profile == :'weather/agriculture_heating')"
+          "hydrogen && (hydrogen.group == :'weather/agriculture_heating')"
```

Comment:
This does not work. The hydrogen burner node in agriculture has no `weather/agriculture_heating` group, but `unmet-demand: useable_heat_from_electricity_curve`. Is there a more generic way to query this? For example something like the following may be more robust:

```
FILTER(
  SECTOR(agriculture),
   "hydrogen && useable_heat_output_conversion > 0.0"
)
```